### PR TITLE
Allow saas file timeouts > 60 minutes

### PR DESCRIPTION
**Pull request by rporres**
APPSRE-4670

Signed-off-by: Rafael Porres Molina <rporres@gmail.com>

### CODE REVIEW

**Review by kwilczynski — APPROVED** [review]
Looks good!

File: schemas/app-sre/saas-file-2.yml [hunk 1]
```diff
@@ -116,6 +116,13 @@ properties:
     description: prevent updates to a saas file
   compare:
     type: boolean
+  timeout:
```
Comment (geoberle) [inline schemas/app-sre/saas-file-2.yml:119]: considering that tenants will most likely not read the schema description, i wonder if we should somehow add the unit to the property name like `timeoutInMinutes`. i know it does not look nice and i'm aware that we have various timeout fields already (most of them without doc string so who knows if it is seconds or minutes or something else) - so i leave this up to you. feel free to ignore.

i would love to have timeouts like these as a string like `timeout: 60m` but we would loose the possibility to check for `minimum` :(

Comment (geoberle) [reply schemas/app-sre/saas-file-2.yml:119]: after having a look at https://github.com/app-sre/qontract-reconcile/pull/2955 i realised that you do additional checks in code as well. that would open up the possibility for the string based version that includes the unit like `pattern: "^\\d+[smhd]$"` and you would still be able to fail PR checks if the value is not considered within the valid range.

Comment (rporres) [reply schemas/app-sre/saas-file-2.yml:119]: It makes sense. It'll make the feature nicer.

Comment (rporres) [reply schemas/app-sre/saas-file-2.yml:119]: Fixed in https://github.com/app-sre/qontract-schemas/pull/312/commits/17fc3eac2abbe1830a3ca47c0b56f066d1102b45

**Review by geoberle — APPROVED** [review]
LGTM

**Pull request merged**